apps/dot-voting/app/components/VotingOption.js

```diff
@@ -18,13 +18,27 @@ const VotingOption = ({ valueSpring, label, percentage, color, threshold, userVo
               {displayLabel}
             </Text>
             {userVote !== -1 && (
-              <Badge
-                foreground={`${theme.info}`}
-                background={`${theme.infoSurface.alpha(0.08)}`}
-                css="margin-left: 10px"
-              >
-                  YOU: {userVote}%
-              </Badge>
+              <div css={`
+                display: inline-flex;
+                justify-content: center;
+                white-space: nowrap;
+                min-width: 24px;
+                width: auto;
+                height: 24px;
+                padding: 1px 12px 0 12px;
+                border-radius: 24px;
+                font-size: 12px;
+                line-height: 1.5;
+                text-transform: uppercase;
+                font-weight: 600;
+                user-select: none;
+                color: ${theme.info};
+                background: ${theme.infoSurface.alpha(0.08)};
+                margin-left: 10px;
+                align-items: center;
```

Comment:
These styles seem pretty similar to `DiskTag` above. Is there a common component we could pull out, that both of these could then start from? It's a lot of styles that seem like maybe they should have a name.